### DISCUSSION TITLE
[Request] adding QMK support for Waterfowl

### DISCUSSION THREAD

**Discussion by Hobbesgoblin** [manna-harbour/miryoku#250] [Q&A] [answered]
Hi.
I wanted to ask if it would be possible to add QMK SUpport for the waterfowl?

Thanks. :)

**Comment by manna-harbour** [comment]
Sure, I'll add it.

(For reference: Waterfowl is here https://github.com/qmk/qmk_firmware/tree/master/keyboards/waterfowl and it looks like the Miryoku ZMK mapping is compatible 
https://github.com/manna-harbour/miryoku_zmk/blob/master/miryoku/mapping/40/waterfowl.h)

**Comment by manna-harbour** [comment, marked as answer]
Added in https://github.com/manna-harbour/miryoku_qmk/commit/2b586786bbbf1a1dd23b2b178b36439cb64fcd51. Sample build run (with `OLED_ENABLE=no` due to AVR space constraints): https://github.com/manna-harbour/miryoku_qmk/actions/runs/6532803747.
